.github/workflows/workflow-validation.yml

```diff
@@ -0,0 +1,29 @@
+name: Workflow Validation Testing
+
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+      - 'feature/**'
+    paths:
+      - '.github/**.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
```

Comment:
Since we don't need `ready_for_review` this should be fine to remove and use the defaults. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
We can simply remove this to use the defaults.